src/plugins/intel_cpu/src/nodes/executors/shl/shl_eltwise.hpp

```diff
@@ -0,0 +1,50 @@
+#include "shl.hpp"
+#include "cpu_memory.h"
+#include "nodes/executors/eltwise.hpp"
+#include <functional>
+#include <tuple>
+#include <utility>
+#include <type_traits>
```

Comment:
Seems like these includes are not necessary. Please verify and remove if it's so